chromium/background.js

```diff
@@ -60,6 +60,11 @@ chrome.storage.onChanged.addListener(async function(changes, areaName) {
       isExtensionEnabled = changes.globalEnabled.newValue;
       updateState();
     }
+    if ('debugging_rulesets' in changes) {
+      const r = new rules.RuleSets();
```

Comment:
This API has changed to requiring `store` gets passed in explicitly. This changed in #12938 

Also, #13213 might land before this, it changes the name of `initialize` to `loadFromBrowserStorage`.